components/components.ignore

```diff
@@ -28,4 +28,4 @@
 # Our build server fails to build the following packages (the builds succeed on other build systems):
 /^runtime\/openjdk-11$/d
 /^runtime\/sbcl$/d
-/^web/icedtea-web/d
+/^web/icedtea-web$/d
```

Comment:
/^web\/icedtea-web$/d

Reply:
That's something where I also easily fail.